src/main/java/com/labsynch/labseer/service/LsThingService.java

```diff
@@ -62,8 +62,13 @@ LsThing saveLsThing(LsThing lsThing, boolean isParent, boolean isBatch,
 
 	Collection<LsThing> findLsThingsByGenericMetaDataSearch(String searchQuery);
 
-	Collection<LsThing> findLsThingsByGenericMetaDataSearch(String lsType,
-			String searchQuery);
+	Collection<LsThing> findLsThingsByGenericMetaDataSearch(String searchQuery,
```

Comment:
I don't see a need for this lsType only route. Also confused that the argument order has been switched.
Is this being used elsewhere in the code? Do we need to switch argument order there as well?

Reply:
Yeah, I also thought it was strange too but it was already there and its only being used in `findLsThingProjectsByGenericMetaDataSearch`.  It works because projects are `lsType:project, lsKind:default`. I suppose the correct fix would be to add an lsKind to that call but I didn't want change too much :
https://github.com/mcneilco/acas-roo-server/blob/1e5dd21e2de0c031c5133863e2aeb93b8a903173/src/main/java/com/labsynch/labseer/service/LsThingServiceImpl.java#L1039

> Also confused that the argument order has been switched.

I switched it to match the way was actually being used in the one place it was being used.

Reply:
Okay, makes sense.